pkg/deployments/deployment-txs/mainnet.json

```diff
@@ -92,5 +92,7 @@
   "0x5Dd94Da3644DDD055fcf6B3E1aa310Bb7801EB8b": "0x39f357b78c03954f0bcee2288bf3b223f454816c141ef20399a7bf38057254c4",
   "0xdba127fBc23fb20F5929C546af220A991b5C6e01": "0xd8c9ba758cb318beb0c9525b7621280a22b6dfe02cf725a3ece0718598f260ef",
   "0x1576d472d82A72a1Ec7C57FcA770BD752D124A62": "0xed8cfdf17294c227023a1083f1d0ee385873a6458a7132e3a251ca4cd33621f0",
-  "0x222bc81C6F3C17e9e9Aba47a12f55a1Dea42f163": "0x83fb41bb8a64dc323255fc3a98d544fbd88a37e6447054c7b4dd16fc6251feac"
+  "0x222bc81C6F3C17e9e9Aba47a12f55a1Dea42f163": "0x83fb41bb8a64dc323255fc3a98d544fbd88a37e6447054c7b4dd16fc6251feac",
+  "0x67A25ca2350Ebf4a0C475cA74C257C94a373b828": "0xf252d24fcb350febba5b414dcdf93a6b08e558e384a19bf88abaf8b0dde58d14",
+  "0xE605Dbe1cA85dCdb8F43CEfA427f3B0fC06f6ba6": "0xb8151a4af682703443a331e769d6d92a04844ea2793b9dcf496811c0158d6d5b"
```

Comment:
Pool seems to be missing (so the verification is pending)

Reply:
I believe he said he ran out of ETH and didn't have enough to deploy the pool. Will look into it